src/Ast.ml

```diff
@@ -225,6 +225,12 @@ and Requires_sub_terms : sig
 
   val exposed : cls -> expression -> bool
 
+  type ltgt = LT | GT
+
+  val typ_exposed_left : ltgt -> core_type -> bool
```

Comment:
nit: the other functions, e.g. parenze_*, have "type" suffixes rather than prefixes